src/test/java/by/innowise/monolithwebcalculator/domain/mapper/OperationMapperTest.java

```diff
@@ -8,21 +8,21 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-
+import org.mapstruct.factory.Mappers;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 
 class OperationMapperTest {
 
-    static OperationMapper mapper;
+    static OperationMapper operationMapper;
```

Comment:
You can use `@TestInstance` annotation to make `@BeforeAll` method be non static. Do not declare static class members instead.